client/src/initial/landingpagetanu/tanulanding.scss

```diff
@@ -0,0 +1,120 @@
+.page-container {
```

Comment:
for all classes please rename the classes so that it's more unique, this will just ensure that your newly defined classes don't affect other components (that might have the same classname).

for example, you can add the prefix `landing-page-tanu-` to all classes since we have multiple landing page components.